shared-spring/src/main/java/tds/shared/spring/interceptors/RestTemplateLoggingInterceptor.java

```diff
@@ -37,7 +37,7 @@ public ClientHttpResponse intercept(final HttpRequest request,
                                         final ClientHttpRequestExecution clientHttpRequestExecution) throws IOException {
 
         // get tracer id that used to associate all rest calls that are a child of this request
-        final String traceId = Optional.fromNullable(MDC.get(TRACER_ID_HEADER)).or(UUID.randomUUID().toString());
+        final String traceId = Optional.fromNullable(MDC.get(TRACER_ID_HEADER)).or(UUID.randomUUID().toString().replace("-", ""));
```

Comment:
The microservices do not generate trace ids in user code, it is done by the sleuth code when needed.